backend/utils.py

```diff
@@ -175,18 +175,28 @@ def offset_pagination(page, number_fetchs, query):
     return [query, more]
 
 
-def paginate(params, query):
-    """Paginate events."""
-    params = dict(params)
+def query_paginated(request_data, query, page=None, limit=None):
```

Comment:
page=Utils.DEFAULT_PAGINATION_OFFSET

That already assign the default value to the variable if None is coming from the call.